### PR TITLE
Add captcha providers / update templates

### PR DESCRIPTION
**Pull request by LaurenceJJones**
Add turnstile and hcaptcha as other providers.

Within haproxy server class we cannot see the DNS name only the resolved IP. So we have to get the user to specify which provider it is inside the bouncer configuration. Another way would be to have a list of "hardcoded" known server names EG "hcaptcha_verifier" and once we see this server name we know they want the hcaptcha configuration.

### CODE REVIEW

**Comment by LaurenceJJones** [comment]
Converting to draft as I want to test if my new detection of captcha provider works before making it open again

**Comment by LaurenceJJones** [comment]
@ciphermenial completed it. Turns out the code different as haproxy has different internal libraries. I have tested all captchas, I need to document how to choose the resolver as its a bit magic 🧙🏻 .

File: install.sh [hunk 1]
```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 
-LUA_MOD_DIR="./lua-mod"
-LIB_PATH="/usr/local/crowdsec/lua/haproxy/"
+LUA_MOD_DIR="."
```
Comment (nitescuc) [inline install.sh:3]: Could you please just double check with @sabban about where those files should be to understand who of code or documentation is wrong (if not both :D )

**Review by nitescuc — APPROVED** [review]
approved, just check this path thing please

**Pull request merged**